### PR TITLE
Grinder Fix + Updates

### PR DESCRIPTION
**Pull request by ArcLumin**
Adds in coin grinding, butt grinding for fartium, cig grinding for nicotine, bible grinding for holy water, rainbow crayon grinding for colorful reagent, increases crayon powder amounts. Fixes grinder being unable to grind food.

### CODE REVIEW

**Comment by GenericGoose** [comment]
fartium is op.
grind 2 butts you get 40u of fartium, which metabolizes at rate 0.2u/tick,will metabolize in 200 ticks.
regularly, if you have no ass it deals 1 toksin damage, however...
30u is overdose threshold, you'll be taking 2 toksin damage
add 1 more butt and you'll be addicted (50u), in which case you'll be taking from 3 to 6 toksin damage.
also didn't i fiks grinder by reverting something someone did to the istype proc

**Comment by ArcLumin** [comment]
Nope, it's still broken

And if we're talking about OP chems: Plasma will kill in 15u and you can get tons of it by grinding plasteel. You've got loads of chems that kill in WAY less than that, too. The issue is "How you get it to the person"; I mean, getting anyone addicted to anything is insanely strong, and fartium addiction isn't nearly as strong as the others. And almost everything with an addiction has an addiction of far lower, and even stronger overdose damage on just about everything with an overdose. When you compare it to other chems we have now, it's not OP.

**Comment by GenericGoose** [comment]
yeah eksept fartium is the easiest to get

**Comment by ArcLumin** [comment]
Not really. To cause the addiction process, you would need 3 butts, meaning you'd have to find and hunt down 2 butts superfarted off of other people, which is not easily done at roundstart, as well as superfarting your own butt off, which you'll need to fix the nutriment loss pretty quick. Meanwhile, you can easily make meth, epinephrine, etc. in far less time, and with significantly better results. Not to mention only the bar, kitchen, chem and science have grinders, and they all have significantly better ways to kill people far faster than trying to shoot them with 4 syringes without any of them missing, multiple smoke grenades which are very conspicuous and people can just walk out of them, etc. All in all, you'd be better off doing an overdose process on just about anything else, hell, even a tricord overdose is far easier to do, far faster to get lots of, and more deadly.

Not even talking about even more deadly chems like lexorin, which kill in far less. And any sheet of plasteel, which can be found everywhere now, has 20u plasma; a lethal amount.

**Comment by Kierany9** [comment]
addiction is implemented terribly

lets remove it

**Comment by Carbonhell** [comment]
goose why the hell do you write the x as ks

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 1]
```diff
@@ -1096,7 +1104,7 @@
 				src.updateUsrDialog()
 				return 0
 
-		if (!is_type_in_list(O, blend_items) && !is_type_in_list(O, juice_items))
+		if (!is_type_in_list(O, blend_items) && !is_type_in_list(O, juice_items) && !istype(O,/obj/item/weapon/reagent_containers/food))
```
Comment (Carbonhell) [inline code/modules/reagents/Chemistry-Machinery.dm]: ???

Comment (ArcLumin) [inline code/modules/reagents/Chemistry-Machinery.dm]: Grinders were still not accepting food, so by forcing it into being accepted, it should be able to grind now. Everything I was seeing is that if you could get them into the grinder by varediting, it'd work, but it wouldn't actually go in, so this is my shit fix

**Comment by Gandalf2k15** [comment]
Did you test this @ArcLumin?

**Comment by ArcLumin** [comment]
Yes, it's tested, and it fixes the inability to put food in. http://puu.sh/pTupe.png

**Comment by ghost** [comment]
Technically feature PR with new grindan stuff but it's fine imo +1.

**Comment by ArcLumin** [comment]
Fixed cigars to put in all their reagents when grinded, fixed pills and cigar grinding as well, since that wasn't included in food.

**Comment by ArcLumin** [comment]
Oh god, the grinder issues are far more far reaching than I thought. It seems microwaves and similar things are having issues too. Seems like something a bit higher up got fucked somewhere down the road.

**Comment by Gandalf2k15** [comment]
Code looks good, check seemed a bit snowflakey at first, however its likely the best way to do it. Just needs head input now.

**Comment by Spacedong** [comment]
@nebyab head input blease

**Comment by Gandalf2k15** [comment]
@headmins input please.

**Comment by ArcLumin** [comment]
@anyone input please

**Comment by calibraptor** [comment]
+1

**Comment by ArcLumin** [comment]
Where's that last +1 lads

**Comment by Nebyab** [comment]
ok

**Comment by Gandalf2k15** [comment]
Cool, head input gained. Merging, already code checked.

**Comment by GenericGoose** [comment]
Sorry i failed to fiks grinder

**Pull request merged**